apps/ejabberd/src/ejabberd.app.src

```diff
@@ -40,7 +40,8 @@
                   usec,
                   uuid,
                   xmerl,
-                  worker_pool
+                  worker_pool,
+                  hackney
```

Comment:
`fusco` seems to be harder to use. It gives me a connection, while `hackney` gives me pool of connections, so I don't have to make my own. 

Also from `fusco's` github: *"Fusco is still in a very early stage of development, be aware of that!*". While `hackney` is respected, good and actively maintained project.